src/pages/library/RequestDatasetAccessModal.js

```diff
@@ -12,111 +12,104 @@ import * as Utils from 'src/libs/utils'
 
 
 const sendCopyEnabled = false
+const requestAccessEnabled = false
 
 export const RequestDatasetAccessModal = ({ onDismiss, datasets }) => {
   const [reason, setReason] = useState('')
   const [sendCopy, setSendCopy] = useState(false)
+  const [showWipModal, setShowWipModal] = useState(false)
 
-  return h(Modal, {
-    title: 'Request Access',
-    width: '40rem',
+  return showWipModal ? h(Modal, {
+    title: `Keep in mind we're still in BETA`,
+    width: '32rem',
     showButtons: false,
-    showX: true,
-    onDismiss
+    onDismiss: () => {}
   }, [
-    div([
-      'You cannot access this dataset because it is protected by an Authorization Domain. ',
-      'You need to obtain permission from the owner(s) of this dataset in order to get access. ',
-      'Clicking the "Request Access" button below will send an email to the admins of that dataset.'
+    div({ style: { lineHeight: '1.7rem' } }, [
+      `In the near future, when you click on `, strong(['Request Access']),
+      `, the request will be sent to the appropriate approver and the dataset will be marked as `,
+      strong(['Pending']), `.`
     ]),
-    h(IdContainer, [id => div([
-      h(FormLabel, { htmlFor: id }, ['Please tell us why']),
-      h(TextArea, {
-        id,
-        style: { height: 100, marginBottom: '0.5rem' },
-        placeholder: 'Enter your reason',
-        value: reason,
-        onChange: setReason
-      }),
-      sendCopyEnabled && div([
-        h(LabeledCheckbox, {
-          checked: sendCopy,
-          onChange: setSendCopy
-        }, [
-          label({ style: { margin: '0 2rem 0 0.25rem' } }, [`Also send me a copy`])
-        ])
-      ])
-    ])]),
-    table({ style: { margin: '1rem', width: '100%' } }, [
-      thead([
-        tr({ style: { height: '2rem' } }, [th({ style: { textAlign: 'left' } }, ['Datasets']), th({ style: { textAlign: 'left', width: '15rem' } }, ['Access'])])
+    div({ style: { marginTop: 30, display: 'flex', flexDirection: 'row', justifyContent: 'space-between', alignItems: 'center' } }, [
+      `Let's continue testing the BETA Data Catalog!`,
+      h(ButtonPrimary, {
+        onClick: onDismiss,
+        style: { padding: '20px 30px' }
+      }, 'OK')
+    ])
+  ]) :
+    h(Modal, {
+      title: 'Request Access',
+      width: '40rem',
+      showButtons: false,
+      showX: true,
+      onDismiss
+    }, [
+      div([
+        'You cannot access this dataset because it is protected by an Authorization Domain. ',
```

Comment:
I'm not super familiar with the hyperscript helpers lib but do we not need indexes for child elements?  Wondering if maybe it adds them when the children are text

Reply:
We have to uniquely key them if the object is part of a list, but otherwise, it's okay!